apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -137,13 +137,20 @@ defmodule SiteWeb.ScheduleController.TimetableController do
 
   @spec trip_schedule(Schedules.Schedule.t()) ::
           {{Schedules.Trip.id_t() | nil, Stops.Stop.id_t()}, Schedules.Schedule.t()}
```

Comment:
I think the spec needs to be updated to include the `| nil` for the `stop.id` return as well

Reply:
Yeah, that's right. Thanks for catching - done